src/components/AddPlaidBankAccount.js

```diff
@@ -155,14 +156,26 @@ class AddPlaidBankAccount extends React.Component {
         });
     }
 
+    /**
+     * @returns {String}
+     */
+    getPlaidLinkToken() {
+        if (!_.isEmpty(this.props.plaidLinkToken)) {
+            return this.props.plaidLinkToken;
+        }
+
+        if (this.props.receivedRedirectURI) {
+            return this.props.plaidLinkOAuthToken;
+        }
+    }
+
     render() {
         const accounts = this.getAccounts();
         const options = _.map(accounts, (account, index) => ({
             value: index, label: `${account.addressName} ${account.accountNumber}`,
         }));
         const {icon, iconSize} = getBankIcon(this.state.institution.name);
         const hasPlaidLinkToken = !_.isEmpty(this.props.plaidLinkToken) || !_.isEmpty(this.props.plaidLinkOAuthToken);
```

Comment:
I think we can simplify this logic by just doing 

```js
const token = this.getPlaidLinkToken();
...
{(!token || this.props.plaidBankAccounts.loading)
                && (
...
{token && (
                    <PlaidLink
                        token={token}
...
```

doesn't seem like `hasPlaidLinkToken` is required...

Basically, if we have a token we can and should use then let's use it otherwise don't render `PlaidLink` at all.